src/async-wrap.cc

```diff
@@ -207,25 +209,23 @@ Local<Value> AsyncWrap::MakeCallback(const Local<Function> cb,
   }
 
   if (ran_init_callback() && !pre_fn.IsEmpty()) {
-    try_catch.SetVerbose(false);
```

Comment:
Also trying to reproduce that for `node::MakeCallback` with the following code:
```
const asyncHooks = process.binding('async_wrap');
const domain = require('domain');
const fs = require('fs');

asyncHooks.setupHooks(_ => _, function () {
  throw new Error('throw in async pre hook');
}, function () {
  throw new Error('throw in async post hook');
});
asyncHooks.enable();

const d = domain.create();

d.on('error', function onError(err) {
	console.log('in domain error handler:', err);
});

d.run(function() {
	setImmediate(function() {
		throw new Error('throw from setImmediate callback');
	});
});
```
doesn't seem to trigger async hooks. Am I missing something?